internal/proxyproto/test_grpc_server.go

```diff
@@ -0,0 +1,89 @@
+package proxyproto
+
+import (
+	"context"
+)
+
+type ProxyGRPCTestServer struct {
```

Comment:
Better to move this to `proxy` package and make non-public, `proxyproto` is for protocol definitions mostly

Reply:
Done